interview-questions.md

```diff
@@ -7,90 +7,89 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method `super()`?
 
-  Your answer:
+  Your answer: super() is used for a new constructor to call on a different class and gain access to that classes constructors.
 
-  Researched answer:
+  Researched answer: Calls the constructor from the parent class and gives us access to the parent classes properties and methods
 
 
 
 2. What is the DOM? What is the virtual DOM?
 
-  Your answer:
+  Your answer: Not sure what it stands for, but I think it basically represents an object on an html page as well as its javascript and css. 
 
-  Researched answer:
+  Researched answer: Stands for Document Object Model. It is an interface that allows functions to dynamically change anything on a document/webpage.
 
 
 
 3. What is a higher-order function?
 
-  Your answer:
+  Your answer: It's different from a built in function, higher-order functions can be attached to an object or another method and follows a set syntax inside its parenthesis that will return the outcome with your own twist.  
 
-  Researched answer:
+  Researched answer: They take functions that take other functions as arguments. They can also return functions as return values, or both.
 
 
 
 4. What is JSX?
 
-  Your answer:
+  Your answer: A combination of javascript and html so that you don't need 2 separate files.
```

Comment:
Totally, but I'd think it might be better to say something like "It stands for Javascript XML, it allows you to use HTML within your Javascript."